apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -51,7 +52,7 @@ export const blocks = GoogleBlockly.common.createBlockDefinitionsFromJsonArray([
     helpUrl: '%{BKY_PROCEDURES_DEFNORETURN_HELPURL}',
     tooltip: '%{BKY_PROCEDURES_DEFNORETURN_TOOLTIP}',
     extensions: [
-      'procedure_def_get_def_mixin',
+      'cdo_procedure_def_get_def_mixin',
```

Comment:
Do you think there's potential confusion about the use of "CDO" here since that's often used to refer to CDO vs. mainline Blockly? Would `local_procedure_def_get_def_mixin` or `custom_procedure_def_get_def_mixin` make it clearer that you're referring to a local copy of the mixin?